mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -345,6 +341,31 @@ func resourceMongoDBAtlasAdvancedCluster() *schema.Resource {
 	}
 }
 
+var tagsSchema = schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:     schema.TypeString,
+				Optional: true, // TODO verify if truly optional
```

Comment:
From local executions I was able to see that both key and value fields are required, will check in mms and adjust API Spec if needed.